content/docs/sidebar.json

```
@@ -53,10 +53,14 @@
         }
       },
       {
-        "slug": "experiments",
+        "label": "Metrics, Parameters, and Plots",
+        "slug": "metrics-parameters-plots",
         "tutorials": {
           "katacoda": "https://katacoda.com/dvc/courses/get-started"
         }
+      },
+      {
+        "slug": "experiments"
       }
```

Just this may work? Forgot
```suggestion
      "experiments"
```

---

Sadly it does not!